walk-in-interview/src/main/java/com/google/job/data/JobsDatabase.java

```diff
@@ -0,0 +1,68 @@
+package com.google.job.data;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.cloud.firestore.Firestore;
+import com.google.firestore.FireStoreUtils;
+
+import java.util.concurrent.ExecutionException;
+
+/** Interaction methods between job posts and database. */
```

Comment:
Nit: maybe add "helps persist and retrieve jobs" ? Not sure if job post has "interaction" with db?